src/fswAlgorithms/attGuidance/oneAxisSolarArrayPoint/oneAxisSolarArrayPoint.h

```diff
@@ -29,7 +29,7 @@
 #include "cMsgCInterface/EphemerisMsg_C.h"
 #include "cMsgCInterface/NavAttMsg_C.h"
 
-#define EPS 1e-12
```

Comment:
make the EPS tolerance less small, maybe 1e-12?

Reply:
Done